src/main/java/seedu/application/logic/Messages.java

```diff
@@ -14,10 +14,11 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown Command.\n" + HelpCommand.MESSAGE_USAGE;
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s\n";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n\n%1$s\n";
     public static final String MESSAGE_INVALID_JOB_DISPLAYED_INDEX = "The job index provided is invalid!";
-    public static final String MESSAGE_INVALID_SPECIFIER = "The field specifier provided is invalid!";
-    public static final String MESSAGE_JOBS_LISTED_OVERVIEW = "%1$d jobs listed!";
+    public static final String MESSAGE_INVALID_PREFIX = "The prefix provided is invalid!";
```

Comment:
Can I clarify the difference between the meaning btw "Prefix" and "Field Specifier"? It is correct if I understand as this:
- Prefix is targeted for the field -> c/ r/ etc
- Field Specifier is for the command -> -c -r etc

so technically if I specifier -c-r in the command input, I must have c/ r/ in the subsequent entry of the command?